agent/config/config_windows.go

```diff
@@ -59,7 +60,11 @@ func DefaultConfig() Config {
 			netBIOSPort,
 		},
 		ReservedPortsUDP: []uint16{},
-		DataDir:          filepath.Join(ecsRoot, "data"),
+		DataDir:          dataDir,
+		// DataDirOnHost is identical to DataDir for Windows because we do not run on a container
+		// We use this redundancy to avoid making significant changes between the Windows and Linux
```

Comment:
I think the first sentence is fine, but the next one is unnecessary.  It's sufficient to say that it's the same because we're not running inside a container.